tests/test_eflyt/test_case.py

```diff
@@ -0,0 +1,77 @@
+"""Tests related to the Case module"""
+
+import os
+import unittest
+from datetime import date
+
+from dotenv import load_dotenv
+from selenium.webdriver.common.by import By
+
+from itk_dev_shared_components.eflyt.eflyt_login import login
+from itk_dev_shared_components.eflyt.eflyt_search import open_case
+from itk_dev_shared_components.eflyt import eflyt_case
+
+load_dotenv()
+test_cpr = os.getenv("TEST_CPR")
+test_case = os.getenv("TEST_CASE")
+test_no_inhabitants_case = os.getenv("TEST_CASE_NOONE")
+
+
+class CaseTest(unittest.TestCase):
```

Comment:
Test classes are normally named "TestFooBar" in our projects

Reply:
Even though "TestCase" might be confusing...

Reply:
Look at me breaking my own conventions. You're right... I have introduced new tests that follow TestFooBar because I dum dum